### PR TITLE
[WIP][DNM] Moves some admin verbs around in various permission groups.

### PR DESCRIPTION
**Pull request by Shadowlight213**
FOR THE LOVE OF GOD DON'T MERGE THIS UNTIL MSO GIVES AN OK.
HEAVILY WIP. I haven't looked for any in verb checks that lock stuff to a permission. this currently is only groups so we can sort stuff out.

Basically, the idea is to look at updating the permission flag requirements for stuff and seeing what we should give to what roles.
FEEDBACK WANTED!!
MSO and headmin feedback obviously is required for this to go beyond theoretical, but I also would like the feedback of everyone with the coder rank as well.
@MrStonedOne @lollerderby

### CODE REVIEW

**Comment by tkdrg** [comment]
shadowlight asked me to ping @tgstation/commit-access

**Comment by duncathan** [comment]
>tfw the admins don't trust you even a little bit

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -89,11 +87,13 @@ var/list/admin_verbs_fun = list(
 	/client/proc/forceEvent,
 	/client/proc/bluespace_artillery,
 	/client/proc/admin_change_sec_level,
-	/client/proc/toggle_nuke
+	/client/proc/toggle_nuke,
+	/client/proc/toggle_AI_interact /*toggle admin ability to interact with machines as an AI*/
```
Comment (neersighted) [inline code/modules/admin/admin_verbs.dm]: Too useful for debugging (tgui) issues without spawning self

Comment (optimumtact) [inline code/modules/admin/admin_verbs.dm]: it's in the debug list anyway

**Comment by Shapsy** [comment]
Why lock the AI interaction feature behind FUN/DEBUG? It's a very useful tool and trialmins shouldn't have it locked away.

**Comment by MrStonedOne** [comment]
Nothing offensive there.

**Comment by Shadowlight213** [comment]
@Shapsy I can move it back to +ADMIN. It just looked like something that belonged in the +FUN category to me.

**Comment by Dorsisdwarf** [comment]
Remember that +FUN isn't just for everything !FUN!, it's more like, for crazy messing around stuff and shortcuts for the lazy, IIRC.

**Comment by Cheridan** [comment]
stale pr, reopen by request

**Pull request closed without merging**